docs/user/experimental/keyboard/modal.md

```diff
@@ -59,6 +59,7 @@ To bind keys that do not have modifiers (eg just pressing `a`), add entries like
     "ss": "boundedNonWhitespaceSequence",
     "sa": "argumentOrParameter",
     "sl": "url",
+    "vz": "visible",
```

Comment:
The `v` key is already used for the `curve` shape below, so this will need to use a different key sequence.